lib/rubocop/cop/betterment/implicit_redirect_type.rb

```diff
@@ -46,15 +46,15 @@ class ImplicitRedirectType < Cop
         def on_block(node)
           return unless routes_file?
 
-          if block_form_with_options(node) { |options| options.none?(&method(:valid_status_option?)) } || block_form_without_options?(node)
+          if block_form_with_options(node) { |options| options.none? { |n| valid_status_option?(n) } } || block_form_without_options?(node)
```

Comment:
This is a performance cop - apparently `&method(...)` is much slower, which is not good for linting purposes.